exporter/exporterbatcher/batch_func.go

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package exporterbatcher // import "go.opentelemetry.io/collector/exporter/exporterbatcher"
+
+import "context"
+
+// BatchMergeFunc is a function that merges two requests into a single request.
+// Do not mutate the requests passed to the function if error can be returned after mutation or if the exporter is
+// marked as not mutable.
+// This API is at the early stage of development and may change without backward compatibility
```

Comment:
Can you put an `Experimental` at the start? That way we can easily grep for experimental APIs (e.g. when we work on stabilizing the `exporterhelper` module

```suggestion
// Experimental: This API is at the early stage of development and may change without backward compatibility
```

Reply:
Make sense. Updated 👍